app/Console/Commands/MigrateCouch.php

```diff
@@ -0,0 +1,129 @@
+        // }
+
+        $clientUsers = CouchDbHelper::client('_users');
+        $adm = CouchDbHelper::admin();
+
+        $this->info('✓ Removing every couchdb database');
+        $dbs = $clientUsers->listDatabases();
+        foreach ($dbs as $dbName) {
+            $db = CouchDbHelper::client($dbName);
+            $db->deleteDatabase();
+        }
+        $this->info('✓ Creating _users database');
+        $clientUsers->createDatabase();
+
+        /**
+         * Users.
+         */
+        $users = DB::table('users')->get();
+        $this->info('✓ Migrating '.count($users).' users');
+        $output = new ConsoleOutput();
+        $progress = new ProgressBar($output, count($users));
+        $progress->start();
+        foreach ($users as $user) {
+            CouchUser::createFromUser($user);
+            $progress->advance();
+        }
+        $progress->finish();
+        $this->info('');
+        $this->info('');
+
+        /**
+         * Accounts.
+         */
+        $accounts = DB::table('accounts')->get();
+        $this->info('✓ Migrating '.count($accounts).' accounts');
+        $output = new ConsoleOutput();
+        $progress = new ProgressBar($output, count($accounts));
+        $progress->start();
+        foreach ($accounts as $account) {
+            // create its database
+            $client = CouchDbHelper::getAccountDatabase($account->id);
+            $client->createDatabase();
+            CouchDbHelper::admin(CouchDbHelper::getAccountDatabaseName($account->id))->addDatabaseMemberRole(CouchDbHelper::getAccountDatabaseRoleName($account->id));
+
+            //setting note design documents
+            $favNotesFn = "function(doc) { if (doc.type === 'note' && doc.is_favorited === true) { emit(doc.created_at, null); } }";
+            $notesByContact = "function(doc) { if (doc.type === 'note') { emit([doc.contact_id, doc.created_at], null); } }";
+            $design_doc = (object) [
+                '_id' => '_design/notes',
+                'language' => 'javascript',
+                'views' => [
+                    'favorites'=> ['map' => $favNotesFn],
+                    'byContact' => ['map' => $notesByContact],
+                ],
+            ];
+            $client->storeDoc($design_doc);
+            $progress->advance();
+        }
+        $progress->finish();
+        $this->info('');
+        $this->info('');
+
+        $notes = DB::table('notes')->get();
+        $this->info('✓ Migrating '.count($notes).' notes');
+        $output = new ConsoleOutput();
+        $progress = new ProgressBar($output, count($notes));
+        $progress->start();
+        foreach ($notes as $note) {
+            $client = CouchDbHelper::getAccountDatabase($note->account_id);
+
+            $new_doc = (object) [
+                '_id' => 'note-legacy-'.$note->id,
+                'type' => 'note',
+                'contact_id' => $note->contact_id,
+                'created_at' => $note->created_at,
```

Comment:
In theory `$note->created_at` returns a Carbon instance. Does it store a string here or the Object?

Reply:
It stores a string here. Do we need to store timezone here too ? Can't we use GMT timezone everywhere ?